generators/ci-cd/index.js

```diff
@@ -193,6 +193,28 @@ module.exports = class extends BaseBlueprintGenerator {
     return this._configuring();
   }
 
+  loadAppConfig(config = _.defaults({}, this.jhipsterConfig, this, defaultConfig), dest = this) {
+    super.loadAppConfig(config, dest);
+  }
+
+  loadClientConfig(config = _.defaults({}, this.jhipsterConfig, this, defaultConfig), dest = this) {
+    super.loadClientConfig(config, dest);
+  }
+
+  loadServerConfig(config = _.defaults({}, this.jhipsterConfig, this, defaultConfig), dest = this) {
+    super.loadServerConfig(config, dest);
+  }
+
+  loadPlatformConfig(config = _.defaults({}, this.jhipsterConfig, this, defaultConfig), dest = this) {
+    super.loadPlatformConfig(config, dest);
+    dest.cicdIntegrationsSnyk = config.cicdIntegrations.includes('snyk');
+    dest.cicdIntegrationsSonar = config.cicdIntegrations.includes('sonar');
+    dest.cicdIntegrationsHeroku = config.cicdIntegrations.includes('heroku');
+    dest.cicdIntegrationsDeploy = config.cicdIntegrations.includes('deploy');
+    dest.cicdIntegrationsPublishDocker = config.cicdIntegrations.includes('publishDocker');
+    dest.cicdIntegrationsCypressDashboard = config.cicdIntegrations.includes('cypressDashboard');
+  }
```

Comment:
A api method in a generator must be prefixed with `_` otherwise it will be queued as a task at default priority.

Reply:
Sorry, but I don't get this one: of course they will be queued, that was what I expected when writing

Reply:
Ok, it would be cleaner to create a priority, can be confusing otherwise, probably the loading priority is the correct one to use.

And drop the parameters, the generator receives the cli command as task arguments.
`jhipster ci-cd foo` would pass ['foo'] as loadPlatformConfig argument.